### PR TITLE
Fix schemas to work with xjc

### PR DESCRIPTION
**Pull request by nvioli**
See [issue 425](https://github.com/IATI/IATI-Schemas/issues/425)

### CODE REVIEW

**Comment by andylolz** [comment]
This change looks good to me. Is there a plan to review and merge?

**Comment by Ocre42** [comment]
This looks like a sensible thing to do, but we're currently not sure how it would affect other people's bespoke systems, so it may be a good idea to bring this up to the community before reaching a decision on our end.

File: iati-organisations-schema.xsd [hunk 1]
```diff
@@ -793,7 +767,6 @@
               <xsd:anyAttribute processContents="lax" namespace="##other"/>
             </xsd:complexType>
           </xsd:element>
-          <xsd:any namespace="##other" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>
```
Comment (andylolz) [inline iati-organisations-schema.xsd:796]: This line is a bugfix.

**Comment by andylolz** [comment]
> This looks like a sensible thing to do, but we're currently not sure how it would affect other people's bespoke systems, so it may be a good idea to bring this up to the community before reaching a decision on our end.

Sounds good. I’ve [posted about this on IATI discuss](https://discuss.iatistandard.org/t/minor-v2-03-schema-refactor-plus-namespace-bugfix/1756).

**Comment by samuele-mattiuzzo** [comment]
I'm merging this as it has been verified as well by me and @amy-silcock and the discuss consultation hasn't gone anywhere

**Review by samuele-mattiuzzo — APPROVED** [review]

**Pull request merged**